packages/kmod-6.12-nvidia-r570/kmod-6.12-nvidia-r570.spec

```diff
@@ -115,9 +120,10 @@ Requires: %{name}
 %package grid
 Summary: NVIDIA %{tesla_major} GRID driver
 Version: %{tesla_ver}
-License: MIT AND GPL-2.0-only
+License: MIT AND GPL-2.0-only AND LicenseRef-NVIDIA-AWS-GRID-EULA
 Requires: %{_cross_os}variant-platform(aws)
 Requires: %{name}
+Requires: %{_cross_os}libstdc++
```

Comment:
This isn't present for the r580 driver. Is it required here and not required there?

Reply:
Thanks for the catch! I rebased the changes and dropped this in the shuffle for 580. It should be required for it as well.